src/gromacs/nblib/topology.h

```diff
@@ -97,6 +99,12 @@ class Topology
     //! Returns exclusions in proper, performant, GROMACS layout
     const gmx::ListOfLists<int>& getGmxExclusions() const { return exclusions_; }
 
+    //! Returns a vector of particles partial charges
```

Comment:
This comment seems wrong. Copy/paste error?

Reply:
I think so